src/index.html

```diff
@@ -1,17 +1,159 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" ></meta>
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    > </meta>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" ></meta>
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css"></link>
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <!-- ===============/ personal-info /=============== -->
+
+      <fieldset class="personal-info container">
+        <legend>Personal information</legend>
+        <span class="form-item">
+          <label for="Surname:">Surname:
+            <input autocomplete="off" type="text"
+              name="surname"
+            minlength="2" maxlength="20"
+            >
```

Comment:
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

fix it all over the page according to the checklist